hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -205,10 +205,14 @@ public static void initTableIfNotExists(Configuration conf) throws IOException {
     // Hadoop FileSystem
     FileSystem fs = FSUtils.getFs(basePath, hadoopConf);
     if (!fs.exists(new Path(basePath, HoodieTableMetaClient.METAFOLDER_NAME))) {
+      String partitionField = conf.getString(FlinkOptions.PARTITION_PATH_FIELD, null);
+      String preCombineField = conf.getString(FlinkOptions.PRECOMBINE_FIELD);
```

Comment:
Inline these 2 variables, does the default partition path (empty string) works here ?

Reply:
No, the empty partition field name would not work.

Reply:
Okey, then just inline these two variables, just like the others.